src/Resources/web/app_dev.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+use Contao\ManagerBundle\ContaoManager\Plugin as ManagerBundlePlugin;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+// The check is to ensure we don't use .env in production
```

Comment:
https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/web/index.php#L10

`.env` is meant to load/override environment variables in development or local environment, not in production. Production should use actual environment variables. Also, as you pointed out, we don't need anything from `.env` in prod, so no reason to read and parse a unused file.